tox.ini

```diff
@@ -24,8 +24,7 @@ setenv =
 commands_pre =
     {envbindir}/buildout -nc {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test
 commands =
-    {envdir}/bin/test {posargs:-cv}
-
+    {envbindir}/test {posargs:-cv}
```

Comment:
The template for tox.ini apparently dropped a line break here. It's just cosmetic, but would improve readability.

Reply:
Fixed this myself, see https://github.com/zopefoundation/meta/commit/b1221c3cc309863074148c9b90beac375c7d6ab4. Just pull zope/meta master again.